content/en/docs/tasks/access-kubernetes-api/custom-resource-definition-versioning.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Serve versioned CustomResourceDefinitions
+reviewers:
+- mbohlool
+- sttts
+- liggitt
+content_template: templates/task
+weight: 30
+---
+
+{{% capture overview %}}
+This page explains how to add versioning information to
+[CustomResourceDefinitions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#customresourcedefinition-v1beta1-apiextensions), to indicate the stability
+level of your CustomResourceDefinitions. It also describes how to upgrade an
+object from one version to another.
+
+{{< note >}}
+**Note**: It is not possible to specify different schemas for different versions
+of a CustomResourceDefinition in a backward-compatible way. You cannot remove
+fields, and if you add fields, you are responsible for ensuring that clients can
+tolerate missing fields.
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Make sure your Kubernetes cluster has a master version of 1.11.0 or higher.
+
+* Read about [custom resources](/docs/concepts/api-extension/custom-resources/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Overview
+
+Kubernetes standard API types include a versioning mechanism, so that developers
```

Comment:
actually we contradict ourselves with the note above where we say we do not allow to evolve schemata.